src/components/Explore/ExploreChart.tsx

```diff
@@ -79,17 +80,20 @@ const ExploreChart: FunctionComponent<{
       </Group>
       <Group key="axes" top={marginTop} left={marginLeft}>
         <Styles.GridLines>
-          <Grid
-            xScale={timeScale}
-            yScale={yScale}
-            width={innerWidth}
-            height={innerHeight}
-          />
+          <GridColumns<Date> scale={timeScale} height={innerHeight} />
+          <GridRows<number> scale={yScale} width={innerWidth} />
```

Comment:
`@vx/grid` has a bug that doesn't allow to pass specific values to the scale. It's picking the correct values somehow (I _think_ that the axis component mutates the scale ticks, and then the grid takes it from the scale). I'm going to dig into this.

Reply:
(later)